### PR TITLE
(Part - 1) Add user's web app settings to accounts.susi

### LINKED ISSUE #266: Add user's web app settings to accounts.susi
**Actual Behaviour**
Currently on the logged In page we see 4 icons of iOS, android, web and cms. Clicking on Web takes us to an empty page.

**Expected Behaviour**
When a user clicks on the Web icon, a page with user's web app settings should be displayed.

**Screenshots of the issue**
a1

**Would you like to work on the issue?**
Yes

### PR DESCRIPTION
**Pull request by PrP-11**
Fixes #266 

Changes: 
The work on issue #266 is divided into two parts. Part - 1 is covered in this PR.

**Part - 1**
- Display user's accounts and web client settings.

**Part - 2**
- Enable edit and save changes for settings.
- Implement similar UI as on chat.susi settings.

Surge Deployment Link: http://nondescript-shop.surge.sh/

Screenshots for the change:
settings

### CODE REVIEW

**Comment by hkedia321** [comment]
@PrP-11 For the points 1-4, it works fine when the `prefLanguage` parameter comes from the API. However for some users, this parameter might be missing, so please handle such cases.

**Comment by mariobehling** [comment]
Please use the same style as in chat.susi.ai settings, e.g. left side bar is not gray here.

**Comment by PrP-11** [comment]
@hkedia321 I have fixed the bug. Thanks for pointing that out. I have also removed the warnings and errors of the console. Regarding the nav bar for mobile view, I am working on it and can be fixed in part - 2. Also the dropdown menu is positioned at bottom already for me. Please check that again and if it still persists please let me know about your browser. Thanks.

@DNS-404 Regading the country code, it is given twice on chat.susi also. I myself was confused to whether show it or not, but now I have removed it now. Thanks.

File: src/components/Settings/Settings.react.js [hunk 1]
```diff
@@ -1,36 +1,455 @@
...
+	}
+
+	apiCall = () => {
+		$.ajax({
+			url: url,
+			type:'GET',
+			dataType: 'jsonp',
+			statusCode: {
+				404: function(xhr) {
+					if(window.console) {console.log(xhr.responseText);}
```
Comment (anshumanv) [inline src/components/Settings/Settings.react.js]: Can you please explain about the use case of this log?

File: src/components/Settings/Settings.react.js [hunk 2]
```diff
@@ -1,36 +1,455 @@
...
+					console.log('Error 503: Server not responding!');
+					document.location.reload();
+				}
+			},
+			crossDomain: true,
+			timeout: 3000,
+			async: false,
+			success: function (response) {
+				console.log(response.settings);
```
Comment (anshumanv) [inline src/components/Settings/Settings.react.js]: Can be removed

File: src/components/Settings/Settings.react.js [hunk 3]
```diff
@@ -1,36 +1,455 @@
...
+      marginTop: '20px',
       paddingRight:10,
       textAlign:'center'
     }
+    let currentSetting;
+
+    if(!this.state.dataFetched && cookies.get('loggedIn')){
+      this.apiCall();
+    }
+    if(this.state.dataFetched && cookies.get('loggedIn')){
```
Comment (anshumanv) [inline src/components/Settings/Settings.react.js]: This block can be removed.

File: src/components/Settings/Settings.react.js (hunk 1 above)
Comment (PrP-11) [reply src/components/Settings/Settings.react.js]: Whenever the user settings are not applicable for a user role or the settings are not found on the accounting model then this error occurs. This is implemented the same way as on chat.susi settings. Thanks.

**Comment by PrP-11** [comment]
@anshumanv I will mimic the settings on chat.susi in the part 2 itself. Currently this can be merged since currently the settings page is empty and I will create PR for part 2 as soon as its complete. Thanks.

@hkedia321 @DNS-404 @anshumanv @akshatnitd @Akshat-Jain @AnupKumarPanwar Please review.

**Pull request merged**